src/main.scss

```diff
@@ -1,17 +1,76 @@
-@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
-
-@media all and (min-width:760px) {body {font-size: 20px}}
-@media all and (max-width:760px) {body {font-size: 19px}}
-@media all and (max-width:720px) {body {font-size: 18px}}
-@media all and (max-width:680px) {body {font-size: 17px}}
-@media all and (max-width:640px) {body {font-size: 16px}}
-@media all and (max-width:600px) {body {font-size: 15px}}
-@media all and (max-width:560px) {body {font-size: 14px}}
-@media all and (max-width:520px) {body {font-size: 13px}}
-@media all and (max-width:400px) {body {font-size: 12px}}
-@media all and (max-width:360px) {body {font-size: 11px}}
-@media all and (max-width:330px) {body {font-size: 10px}}
-@media all and (max-width:300px) {body {font-size: 9px}}
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+@media all and (min-width: 760px) {
```

Comment:
i am thoroughly disgusted by this there's probably a nice scss way to do this

Reply:
probably https://sass-lang.com/documentation/at-rules/control/each#with-maps

yeah i shouldve done this in the first place but this is originally from raw css

Reply:
i'll make this

Reply:
Oh I didn't even realize it was counting down by a contant difference. Let me try to fix

Reply:
uh too late. does it look good